async function foo() {
  try {
    await bar();
  } catch (error) {}
}
